user-console/src/main/java/org/pentaho/mantle/client/admin/ContentCleanerPanel.java

```diff
@@ -235,6 +235,7 @@ public void cancelPressed() {
                       AbstractWizardDialog.ScheduleDialogType.SCHEDULER );
               editSchedule.setShowSuccessDialog( false );
               editSchedule.addCustomPanel( scheduleLabelPanel, DockPanel.NORTH );
+              editSchedule.setFocusWidget( scheduleTextBox );
```

Comment:
Like previously requested in https://github.com/pentaho/pentaho-platform/pull/5210#discussion_r1095697303, I believe that you should not need to call `setFocusWidget( . )`.
Is it not the first "tababble" element of the dialog? If the code which sets the default focus is failing, we should fix that instead. See https://github.com/pentaho/pentaho-commons-gwt-modules/blob/wcag/widgets/src/main/java/org/pentaho/gwt/widgets/client/dialogs/DialogBox.java#L204.